common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -1059,6 +1059,21 @@ private AuthenticationConstants() {
          */
         public static final String CLIENT_APP_PACKAGE_NAME = "client_app_package_name";
 
+        /**
+         * String for the indicator whether able to connect the auth service.
+         */
+        public static final String AUTH_SERVICE_CONNECT = "auth_service_connect";
+
+        /**
+         * String of account environment key.
+         */
+        public static final String ACCOUNT_ENVIRONMENT_KEY = "account.environment.key";
+
+        /**
+         * String to return account list from broker.
+         */
+        public static final String BROKER_RESULT_ACCOUNTS = "broker.result.accounts";
```

Comment:
We are using snake casing for the constants, this is not part of `BrokerResult` . How about naming `broker_accounts`